backend/src/Squidex/Config/Authentication/OidcHandler.cs

```diff
@@ -40,5 +40,19 @@ public override Task TokenValidated(TokenValidatedContext context)
 
             return base.TokenValidated(context);
         }
+
+        public override Task RedirectToIdentityProviderForSignOut(RedirectContext context)
+        {
+            if (!string.IsNullOrEmpty(options.OidcOnSignoutRedirectUrl))
+            {
+                var logoutUri = options.OidcOnSignoutRedirectUrl;
```

Comment:
The OpenIdOptions has SignedOutRedirectUri...what is wrong with that?

Reply:
Because it is not the same this. `SignedOutRedirectUri` is the location to send browser after all the logout is done. 

If your External OICD provider does not support logout then the logout url returned during metatdata discovery from oidcMetadataAddress is empty. When the external redirect URL is empty the following exception is thrown even if you specify SignedOutRedirectUri

`InvalidOperationException: Cannot redirect to the end session endpoint, the configuration may be missing or invalid.`

More info in https://stackoverflow.com/questions/49815961/cannot-redirect-to-the-end-session-endpoint-the-configuration-may-be-missing-or

So to fix you need to handle the logout event yourself - hence allowing config to override it here.